package/kedro_viz/models/graph.py

```diff
@@ -497,10 +507,52 @@ def __post_init__(self, data_node: DataNode):
             with dataset._fs.open(load_path, **dataset._fs_open_args_load) as fs_file:
                 self.plot = json.load(fs_file)
 
+        if data_node.is_metric_node():
+            if not dataset._exists() or self.filepath is None:
+                return
+
+            import pandas as pd  # pylint: disable=import-outside-toplevel

```

Comment:
Minor: you should be able to move this disable to the top of the function and then it should apply to everything inside the function. That way you won't have to add it for multiple lines.